txpipe/twopoint_fourier.py

```diff
@@ -231,37 +234,15 @@ def load_maps(self):
             for systmap in systmaps_path.iterdir():
                 try:
                     if systmap.is_file():
-                        if pathlib.Path(systmap).suffix not in [".fits", ".hs"]:
-                            print('Warning: Problem reading systematics map file', systmap)
-                            print('Not a HEALPix .fits or a healsparse .hs file.')
-                            warnings.warn("Systematics map file must be a HEALPix .fits file or a healsparse .hs file.")
+                        if pathlib.Path(systmap).suffix != ".fits":
```

Comment:
I'd suggest to remove this since the suffix can be anything, as long as the file is in fits format, also healpy can read `.fits.gz` files natively. Maybe it's better to do:

```
try:
    if systmap.is_file():
        syst_map = healpy.read_map(systmap_file, verbose=False)
    else:
        warnings.warn(f'Map from {systmap} is missing. Skipping...')
except:
    print(f'A problem occurred while reading {systmap}. Skipping...')
    continue
```

Reply:
Thanks Javi. I'll make the changes.